app/controllers/resources_controller.rb

```diff
@@ -79,6 +79,11 @@ def fix_resources(resources)
       r.addresses.each do |a|
         fix_lat_and_long(a)
       end
+
+      if r.sites.length.zero?
```

Comment:
Sorry for the late, drive-by commenting here, but the idiomatic way to check for an empty array in Ruby is to call the [`.empty?`](https://ruby-doc.org/core-2.7.0/Array.html#method-i-empty-3F) method.